nf_core/pipeline-template/.gitpod.yml

```
@@ -4,8 +4,8 @@ tasks:
     command: |
       pre-commit install --install-hooks
       nextflow self-update
-
+{% if code_linters %}
```

I think this one we can keep, to remove the new line
```suggestion
{%- if code_linters %}
```